assets/css/main.scss

```diff
@@ -1,8 +1,140 @@
+    $side-nav-link-hover: {{ site.data.colors.colors.side-navigation.side-nav-link-hover }};
+    $side-nav-link-hover-bg: {{ site.data.colors.colors.side-navigation.side-nav-link-hover-bg }};
+    $side-nav-link-current: {{ site.data.colors.colors.side-navigation.side-nav-link-current }};
+  {% endif %}
+
+  // Color - Footer
+  {% if site.data.colors.colors.footer.primary-footer-bg %}
+    $primary-footer-bg: {{ site.data.colors.colors.footer.primary-footer-bg }};
+    $primary-footer-link: {{ site.data.colors.colors.footer.primary-footer-link }};
+    $footer-bg: {{ site.data.colors.colors.footer.footer-bg }};
+    $footer-text: {{ site.data.colors.colors.footer.footer-text }};
+    $footer-link: {{ site.data.colors.colors.footer.footer-link }};
+  {% endif %}
+
+  // Color - USA banner
+  {% if site.data.colors.colors.usa-banner.usa-banner-bg %}
+    $usa-banner-bg: {{ site.data.colors.colors.usa-banner.usa-banner-bg }};
+    $usa-banner-text: {{ site.data.colors.colors.usa-banner.usa-banner-text }};
+    $usa-banner-link: {{ site.data.colors.colors.usa-banner.usa-banner-link }};
+    $usa-banner-link-hover: {{ site.data.colors.colors.usa-banner.usa-banner-link-hover }};
+  {% endif %}
+
+  // Color - Hero
+  {% if site.data.colors.colors.hero %}
+    $hero-bg: {{ site.data.colors.colors.hero.hero-bg }};
+    $hero-header-alt: {{ site.data.colors.colors.hero.hero-header-alt }};
+    $hero-header: {{ site.data.colors.colors.hero.hero-header }};
+    $hero-text: {{ site.data.colors.colors.hero.hero-text }};
+    $hero-link: {{ site.data.colors.colors.hero.hero-link }};
+    $hero-button-bg: {{ site.data.colors.colors.hero.hero-button-bg }};
+    $hero-button-text: {{ site.data.colors.colors.hero.hero-button-text }};
+  {% endif %}
+
+  // Color - Alt section
+  {% if site.data.colors.colors.alt-section %}
+    $alt-section-bg: {{ site.data.colors.colors.alt-section.bg-color }};
+    $alt-section-header: {{ site.data.colors.colors.alt-section.header-color }};
+    $alt-section-text: {{ site.data.colors.colors.alt-section.text-color }};
+    $alt-section-link: {{ site.data.colors.colors.alt-section.link-color }};
+    $alt-section-link-hover: {{ site.data.colors.colors.alt-section.link-hover-color }};
+  {% endif %}
+
+{% endif %}
+
 @import 'uswds/all';
+
+{% if site.data.colors.colors %}
+  {% if site.data.colors.colors.usa-banner %}
+     @import 'usa-banner';
+  {% endif %}
+  {% if site.data.colors.colors.header %}
+     @import 'header';
+  {% endif %}
+  {% if site.data.colors.colors.top-navigation %}
+    @import 'top-nav';
+  {% endif %}
+  {% if site.data.colors.colors.footer %}
+    @import 'footer';
+  {% endif %}
+  {% if site.data.colors.colors.alt-section %}
+    @import 'alt-color-section';
+  {% endif %}
+  {% if site.data.colors.colors.side-navigation %}
+    @import 'side-nav';
+  {% endif %}
+  {% if site.data.colors.colors.hero %}
+    @import 'hero';
+  {% endif %}
+
+  @import 'custom';
```

Comment:
Do we want the `custom` import within the `site.data.colors.colors` if statement?

Reply:
I'm thinking outside gives more flexibility